app/views/collections/index.html.erb

```diff
@@ -3,7 +3,7 @@
   <tr>
     <th scope="col">Collection</th>
     <th scope="col">Druid</th>
-    <th scope="col">On/off</th>
+    <th scope="col">Active / inactive</th>
```

Comment:
should both be capitalized?  Do we like the spaces around the slash?

Reply:
Fixed @ndushay